### ISSUE TITLE
Fresh clone does not pass lint

### ISSUE THREAD

**Issue by adamthesax** [video-dev/go-elementalconductor#1]
I'm working on adding b-frame support to `video-dev/video-transcoding-api` so I need to add support to `go-elementalconductor` first

After cloning the repo running `make test` fails with a bunch of lint errors. I'm going to go through and fix the lint errors and get that merged down before I go on to b-frame support.

**Comment by adamthesax** [comment]
After a deeper look, It appears the only two linters are failing are `funlen` and `wsl`, for now I'm going to just disable those two linters in the `Makefile`.

**Comment by adamthesax** [comment]
For record keeping. Here are the errors before I disabled `funlen` and `wsl`:

```
client_test.go:52: Function 'TestDoRequiredParameters' is too long (71 > 60) (funlen)
func TestDoRequiredParameters(t *testing.T) {
job_test.go:54: Function 'TestCreateJob' is too long (97 > 60) (funlen)
func TestCreateJob(t *testing.T) {
job_test.go:154: Function 'TestGetJob' is too long (169 > 60) (funlen)
func TestGetJob(t *testing.T) {
job_test.go:356: Function 'TestCancelJob' is too long (93 > 60) (funlen)
func TestCancelJob(t *testing.T) {
job_test.go:475: Function 'TestVideoInfoDimensions' is too long (63 > 60) (funlen)
func TestVideoInfoDimensions(t *testing.T) {
node_test.go:10: Function 'TestGetNodes' is too long (137 > 60) (funlen)
func TestGetNodes(t *testing.T) {
presets_test.go:64: Function 'TestGetPreset' is too long (142 > 60) (funlen)
func TestGetPreset(t *testing.T) {
presets_test.go:209: Function 'TestGetPresetForHls' is too long (150 > 60) (funlen)
func TestGetPresetForHls(t *testing.T) {
presets_test.go:362: Function 'TestCreatePreset' is too long (129 > 60) (funlen)
func TestCreatePreset(t *testing.T) {
client.go:70:2: only one cuddle assignment allowed before if statement (wsl)
        if err != nil {
        ^
client.go:73:2: assignments should only be cuddled with other assignments (wsl)
        req, err := http.NewRequest(method, c.Host+apiPath, strings.NewReader(string(xmlRequest)))
        ^
client.go:74:2: only one cuddle assignment allowed before if statement (wsl)
        if err != nil {
        ^
client.go:77:2: assignments should only be cuddled with other assignments (wsl)
        expiresTime := time.Now().Add(time.Duration(c.AuthExpires) * time.Second)
        ^
client.go:79:2: only cuddled expressions if assigning variable or using from line above (wsl)
        req.Header.Set("Accept", "application/xml")
        ^
client.go:89:2: defer statements should only be cuddled with expressions on same variable (wsl)
        defer resp.Body.Close()
        ^
client.go:91:2: only one cuddle assignment allowed before if statement (wsl)
        if err != nil {
        ^
client.go:94:2: if statements should only be cuddled with assignments (wsl)
        if resp.StatusCode < http.StatusOK || resp.StatusCode >= http.StatusMultipleChoices {
        ^
client.go:100:2: if statements should only be cuddled with assignments (wsl)
        if out != nil && len(respData) > 1 {
        ^
client.go:103:2: return statements should not be cuddled if block has more than two lines (wsl)
        return nil
        ^
client.go:119:2: return statements should not be cuddled if block has more than two lines (wsl)
        return hex.EncodeToString(hasher.Sum(nil))
        ^
config.go:20:2: return statements should not be cuddled if block has more than two lines (wsl)
        return &config, err
        ^
job.go:68:2: declarations should never be cuddled (wsl)
        var payload = struct {
        ^
job.go:71:2: assignments should only be cuddled with other assignments (wsl)
        err := c.do("POST", "/jobs/"+jobID+"/cancel", payload, &job)
        ^
time.go:56:2: declarations should never be cuddled (wsl)
        var content string
        ^
time.go:61:2: if statements should only be cuddled with assignments (wsl)
        if content == "" {
        ^
client_test.go:54:2: declarations should never be cuddled (wsl)
        var data []byte
        ^
client_test.go:60:2: only one cuddle assignment allowed before defer statement (wsl)
        defer server.Close()
        ^
client_test.go:132:2: only one cuddle assignment allowed before defer statement (wsl)
        defer server.Close()
        ^
config_test.go:19:2: only one cuddle assignment allowed before defer statement (wsl)
        defer server.Close()
        ^
job_test.go:344:2: ranges should only be cuddled with assignments used in the iteration (wsl)
        for _, test := range tests {
        ^
job_test.go:527:3: only cuddled expressions if assigning variable or using from line above (wsl)
                t.Run("", func(t *testing.T) {
                ^
job_test.go:525:2: ranges should only be cuddled with assignments used in the iteration (wsl)
        for _, test := range tests {
        ^
job_test.go:550:2: ranges should only be cuddled with assignments used in the iteration (wsl)
        for _, test := range tests {
        ^
```

**Comment by adamthesax** [comment]
Fixed by #2
